### ISSUE TITLE
filter-none, all

### ISSUE THREAD

**Issue by mechvel** [agda/agda-stdlib#264]
``Data.List.Properties.filter-none``  has a strange mnemonic in its name. It works as 
"filter-though all".  
``filter'' is defined as  filter-through, not as filter-out.
Probably, it is needed renaming  ``filter-none <---> filter-all``.
(?)

**Comment by MatthewDaggitt** [comment]
Yes you're right. All four names need to be flipped. I was getting confused in #234
